modules/changePassword.php

```diff
@@ -0,0 +1,92 @@
+<?php
+if (!$user) {
+    header("location: index.php");
+} else {
+    $user_id = $user["id"];
+    $user_pw = $user["password"];
+}
+
```

Comment:
You don't need to use `else`, let add `exit` below line No.3 `header("location: index.php");`